test-apps/01-sample-app/app/mirage/index.js

```diff
@@ -1,4 +1,5 @@
-import login, { loginUsers } from 'ember-rxid/mirage/login';
-import setupMirageData from 'ember-rxid/mirage/seedData';
+import login from 'ember-rxid/mirage/login';
+
+export { default as setupMirageData } from 'ember-rxid/mirage/seedData';
+export { loginUsers as superUsers } from 'ember-rxid/mirage/login';
 export default login;
```

Comment:
It was just a sample with I posted it, but should prob simplify this. I dont think the login, superUser etc stuff should be here.